metabase/metabase
# propagate combo stacking to area and bar charts

Epic https://github.com/metabase/metabase/issues/41976

### Description

This PR enables creating stacked combo charts for Bar/Area visualizations. It requires removing `Stacked chart type` which allowed to override area/bar chart type: 
<img width="317" alt="Screenshot 2024-05-13 at 10 51 23 PM" src="https://github.com/metabase/metabase/assets/14301985/27286aef-689e-492f-9680-3c79a69d66a3">

It was a convenient way to switch between area/bar appearance although redundant. Now this setting does not make sense since it is possible to have both area and bar stacks on the same chart.

This PR also fixes the issue on non-linear y-axis scales when some series were stacked and others don't.

### How to verify

- Create multiseries stacked bar/area charts
- Change series settings to create mixed area/bar/line charts with stacking
- Ensure it works for all y-axis scale types

### Demo

<img width="1724" alt="Screenshot 2024-05-13 at 10 53 09 PM" src="https://github.com/metabase/metabase/assets/14301985/40030fb0-704a-4160-8d37-05913cf3c8dd">

### Checklist
